lib/std/fmt.zig

```diff
@@ -1695,3 +1717,17 @@ test "positional with specifier" {
 test "positional/alignment/width/precision" {
     try testFmt("10.0", "{0d: >3.1}", .{@as(f64, 9.999)});
 }
+
+test "vector" {
+    const vbool: @Vector(4, bool) = [_]bool{ true, false, true, undefined };
```

Comment:
Wouldn't trying to read the value of an undefined entry be UB?

Reply:
Try 
```zig
pub fn main() void {
const b: bool = undefined;
warn("{}\n", .{b});
}
```

Reply:
https://en.wikipedia.org/wiki/Undefined_behavior

Reply:
Just fill issue for this behavior.

Reply:
You added a test case to explicitly test undefined behavior. That doesn't make sense to me. It seems like implementing an intentionally fragile test.

Reply:
And again
```zig
pub fn main() void {
const b: bool = undefined;
warn("{}\n", .{b});
}
```
Does that make sense to you?

Reply:
I recommend changing the line to:
```
     const vbool: @Vector(4, bool) = [_]bool{ true, false, true, false }; 
```
Then the test case will make sense.